packages/smooth_app/lib/pages/scan/continuous_scan_page.dart

```diff
@@ -96,20 +81,44 @@ class ContinuousScanPage extends StatelessWidget {
                         )
                       ],
                     ),
-                    Column(
-                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                      children: <Widget>[
-                        Container(
-                          padding: const EdgeInsets.only(top: 32.0),
-                          child: Column(
+                    if (_continuousScanModel.isNotEmpty)
+                      Column(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: <Widget>[
+                          Row(
+                            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
                             children: <Widget>[
-                              ScanPage.getContributeChooseToggle(
-                                _continuousScanModel,
-                                context,
+                              ElevatedButton.icon(
+                                icon: const Icon(CupertinoIcons.clear_circled),
+                                onPressed: () async =>
+                                    _continuousScanModel.clearScanSession(),
+                                label: const Text('clear'),
+                              ),
+                              ElevatedButton.icon(
+                                icon: const Icon(Icons.emoji_events_outlined),
+                                onPressed: () async {
+                                  await _continuousScanModel
+                                      .refreshProductList();
+                                  await Navigator.push<Widget>(
+                                    context,
+                                    MaterialPageRoute<Widget>(
+                                      builder: (BuildContext context) =>
+                                          PersonalizedRankingPage(
+                                        _continuousScanModel.productList,
+                                      ),
+                                    ),
+                                  );
+                                },
+                                label: Text(
+                                    appLocalizations.myPersonalizedRanking),
```

Comment:
```suggestion
                                    appLocalizations.myRanking),
```

"My ranking" is enough, "My personalized ranking" is too long. (needs corresponding change in appLocalizations)